### ISSUE TITLE
Link to municipal boundaries view

### ISSUE THREAD

**Issue by loleg** [cividi/spatial-data-package-platform#29]
When directly linked to a Gemeindescan, the user may not be aware - and yet find it quite useful - to see the municipal boundaries which are displayed in the search result.

**Comment by n0rdlicht** [comment]
I could see at least three options here:

1. adding the outline from "empty Gemeinde" as a dedicated snapshot
1. recommending to add the outline as a layer in the individual snapshot on a case by case basis
1. rethinking the background layer as something taken care of by the platform not the snapshot author

**Comment by loleg** [comment]
Let's do whatever is easiest. All three options sound good to me, (1) sounds like the low hanging fruit. Plus, I think it is a bit confusing to jump straight to a dataset after searching for a Gemeinde - but, maybe I'm just used to my ways.

**Comment by n0rdlicht** [comment]
> Let's do whatever is easiest. All three options sound good to me, (1) sounds like the low hanging fruit. Plus, I think it is a bit confusing to jump straight to a dataset after searching for a Gemeinde - but, maybe I'm just used to my ways.

Agreed. I propose making the existing "virtual" snapshot when no snapshots exists a permanent entry when loading a snapshot linked to a Gemeinde and as entry point from the search. In this we should also think about adding a minimal overlay for these adding the OSM source and potentially a link to the Gemeinde.

@yvess & @ratron: you think this would be easy to add?

**Comment by loleg** [comment]
I've added a comment to the issue description about the boundaries view being perhaps ideally not a snapshot, but a permalink containing the name of the municipality.

**Comment by n0rdlicht** [comment]
> I've added a comment to the issue description about the boundaries view being perhaps ideally not a snapshot, but a permalink containing the name of the municipality.

Currently the canonical name of the dataset is directly taken from the BfS dataset, which has one name per Gemeinde which tends to be in the main language, e.g. `Genève` and `Zürich` and combinations/picking one for `Biel/Bienne` and `Fribourg`. We can slugify them and use that as permalink, yes.

**Comment by ratron** [comment]
I'm not sure I understand what you mean by virtual snapshot. There is already json data in each municipality which is shown if no snapshots are assigned. This data could be extended with a legend.
Regarding the URL: gemeindescan.ch/ABC123/ is already taken. /new-municipality/ could be changed to /g/ or /m/. We could extend the municipality model to include a unique slug.

**Comment by n0rdlicht** [comment]
Was thinking of adding the existing /new-municipality/ view as an entry at the top of the snapshot list on the left to reach that view with or without existing snapshots linked to a Gemeinde as well as you mentioned doing a minimal legend overlay like in a snapshot view with metadata from the model.

**Comment by ratron** [comment]
searching for a municipality can lead to two different views:
https://stage.gemeindescan.ch/de/new-municipality/4201/
https://stage.gemeindescan.ch/de/WMOKRS/
These do not mix well ATM. municipality/ always shows onbording-fake sanpshots and only exists/is linked to if no snapshots are available. ABC123/ has no link to municipality/
 I see two options:

1 .Use both and change both (frontend changes)
- municipality/ needs to show snapshots if available
- ABC123/ needs to show link to municipality/ 

2. Only use ABC123/ (no frontend change)
- create/import snapshots with the json from the municipality

**Comment by n0rdlicht** [comment]
I would vote for Option 1 and take it as a chance to add a source to the municipality outlines as well as establishing those as permalinks.
